tools/backoffice-frontend/var/www/swagger-ui/lib/call-trace.js

```diff
@@ -0,0 +1,66 @@
+function findSession() {
```

Comment:
This is no longer used and can be removed. Is this referenced anywhere? If so, we can remove it.

Reply:
(let's see if we can remove the whole thing, though...)